pkg/controller/secretsyncer/config/agent.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Why isn't this an import, is the fsnotify code not pluggable for our own Load()?

Reply:
That code is entirely copied except for the lint errors I fixed

Reply:
ACJ